src/content/concepts/module-federation.md

```diff
@@ -140,6 +140,49 @@ Each page of a Single Page Application is exposed from container build in a sepa
 
 Many applications share a common components library which could be built as a container with each component exposed. Each application consumes components from the components library container. Changes to the components library can be separately deployed without the need to re-deploy all applications. The application automatically uses the up-to-date version of the components library.
 
+## Dynamic Remote Containers
+
+The container interface supports `get` and `init`.
+`init` is a possible async method that is called with one argument: the share scope object. This object is used as share scope in the remote container and is filled with the provided modules from a host.
+It can be leveraged to connect remote containers to a host container dynamically at runtime
+
```

Comment:
Can you provide an example filename for additional context for the reader in each of the code examples? Would it be something like:
```
__app.js__

```

Reply:
Will do